backend/Setting.ts

```diff
@@ -30,10 +29,11 @@
 import fs from 'fs';
 import os from 'os';
 import path from 'path';
-import pinoLogger from 'pino';
 import { SettingsObj } from '@/types/SettingsObject';
 import { findEtherpadRoot, makeAbsolute } from '@/utils/backend/AbsolutePaths';
 import { argvP } from '@/utils/backend/CLI';
+import log4js from 'log4js';
```

Comment:
why this step backwards?

Reply:
It logs something like this: ✓ Compiled /api/group in 4.6s . I guess the problem is the checkmark as a Unicode character.